app/app.go

```diff
@@ -824,12 +847,9 @@ func New(
 	// can do so safely.
 	moduleGenesisOrder := []string{
 		capabilitytypes.ModuleName, // Must be first.
-
```

Comment:
We should keep these empty lines. They provide grouping and highlight that the ordering is on purpose.